src/shell/Models/ShellExtensions.cs

```diff
@@ -1,10 +1,14 @@
 using System.Diagnostics.CodeAnalysis;
 using Newtonsoft.Json;
+using System.IO.Abstractions;
 
 namespace NeoShell.Models
 {
     class ShellExtensions : List<ShellExtension>
```

Comment:
This should be a static class

Reply:
There are still a few instance methods like Persist, ToJson etc.